build/AvaloniaDependency.props

```diff
@@ -3,14 +3,14 @@
          xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="0.10.18" />
-    <PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.18" />
-    <PackageReference Include="Avalonia.Desktop" Version="0.10.18" />
-    <PackageReference Include="Avalonia.Diagnostics" Version="0.10.18" />
-    <!--<PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="0.10.18" />-->
-    <PackageReference Include="Avalonia.LinuxFramebuffer" Version="0.10.18" />
-    <PackageReference Include="Avalonia.Markup.Xaml.Loader" Version="0.10.18" />
-    <PackageReference Include="Avalonia.ReactiveUI" Version="0.10.18" />
+    <PackageReference Include="Avalonia" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.Controls.DataGrid" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.Desktop" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.Diagnostics" Version="11.0.0-preview4" />
+    <!--<PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.0-preview4" />-->
+    <PackageReference Include="Avalonia.LinuxFramebuffer" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.Markup.Xaml.Loader" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.ReactiveUI" Version="11.0.0-preview4" />
```

Comment:
It seems to be the list of dependencies that will be referenced with the Prism.Avalonia nuget package.
If so, I believe this list is excessive.

Most likely, for the mvvm framework you most likely only need main "Avalonia" package.
Note, that I have used Prism a very little before in WPF, so I might miss some details.

- Avalonia.Desktop - will block application to run only on Win, Mac and Lin. With the way how .NET dependencies work, this dependency unfortunately won't allow users to target Browser (WASM) projects.
- Avalonia.Controls.DataGrid, Avalonia.Diagnostics and Avalonia.Markup.Xaml.Loader - these should be harmless in general, but also bloat application size, when they are not used (that's why  Diagnostics also disable for the release in our templates).
- Avalonia.LinuxFramebuffer - only makes sense if users want to target linux framebuffer, but should be harmless otherwise. 
- Avalonia.ReactiveUI - probably isn't needed here. If Prism uses System.Reactive, this dependency should be added manually (note, Avalonia doesn't reference System.Reactive anymore, starting with coming soon preview5)

Reply:
Some of these packages still are needed for the sample project though.

Reply:
@maxkatz6 in the next preview 5 release of this repo, I'll trim those down. You're right, we should ***only*** be referencing the bare minimum, most of them aren't being used.